src/handlers/__tests__/spid.test.ts

```diff
@@ -0,0 +1,168 @@
+import {
+  successHandler,
+  errorHandler,
+  metadataRefreshHandler,
+  accessLogHandler,
+} from "../spid";
+import mockReq from "../../__mocks__/request";
+import mockRes from "../../__mocks__/response";
+import { task, Task } from "fp-ts/lib/Task";
+import { BlobService } from "azure-storage";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import {
+  aSAMLRequest,
+  aSAMLResponse,
+  aSAMLResponseWithoutRequestId,
+  aSAMLResponseWithoutFiscalCode,
+} from "../../__mocks__/spid";
+import { left, right } from "fp-ts/lib/TaskEither";
+import { some } from "fp-ts/lib/Option";
+import { toError } from "fp-ts/lib/Either";
+
+// Mock an express request and response
+const aMockedRequest = mockReq();
+const aMockedResponse = mockRes();
+
+// Mock access_log module to spy on storeSpidLogs function
+jest.mock("../../utils/access_log", () => ({
```

Comment:
see comment on `access_log` test

Reply:
Done in https://github.com/pagopa/hub-spid-login-ms/pull/28/commits/e9f29a1e8c4b53fa6f1b78857af9daef72bce9e7